### ISSUE TITLE
Small error in doc link

### ISSUE THREAD

**Issue by tylkas** [github/hubot#990]
On https://github.com/github/hubot/blob/master/bin/hubot#L81
```
--- console.error "See https://github.com/github/hubot/blob/master/docs/README.md for more details on getting started."
+++ console.error "See https://github.com/github/hubot/blob/master/docs/index.md for more details on getting started."
```

**Comment by sdimkov** [comment]
:+1:  This looks good to me.

But please make a proper PR instead of inline diff..

**Comment by sdimkov** [comment]
We now have a proper PR for this - #1006

This issue should be closed now.

**Comment by michaelansel** [comment]
Fixed by #1006
